src/ResourceManager/Websites/Commands.Websites.Test/Commands.Websites.Test.csproj

```diff
@@ -44,9 +44,16 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Storage.4.1.0-preview\lib\net45\Microsoft.Azure.Management.Storage.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Websites, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Websites.1.7.1-preview\lib\net452\Microsoft.Azure.Management.Websites.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Management.Websites, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Websites.2.0.0\lib\net452\Microsoft.Azure.Management.Websites.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Please remove these new references as they are found in the common targets.

Reply:
@maddieclayton - we had to include this because during our SDK generation we found a bug  with provisioning code polling code & we were asked to update the Microsoft.Rest.ClientRuntime version you can see the details here https://github.com/Azure/azure-sdk-for-net/pull/4688

I am going to be adding this back since the common targets version is lower & is causing our Set AppServicePlan, test case to fail.